assets/js/versions.js

```diff
@@ -0,0 +1,30 @@
+var versionNodes = [
+    {
+      version: "v2.1.0-alpha.1",
+      url: "https://esl.github.io/MongoosePush/v2.1.0-alpha.1/introduction.html"
+    },
+    {
+      version: "v2.0.2",
+      url: "https://esl.github.io/MongoosePush/v2.0.2/readme.html#content"
+    },
+    {
+      version: "v2.0.1",
+      url: "https://esl.github.io/MongoosePush/v2.0.1/readme.html#content"
+    },
+    {
+      version: "v2.0.0-beta.1",
+      url: "https://esl.github.io/MongoosePush/v2.0.0-beta.1/readme.html#content"
+    },
+    {
+      version: "v2.0.0-alpha.4",
+      url: "https://esl.github.io/MongoosePush/v2.0.0-alpha.4/readme.html#content"
+    },
+    {
+      version: "v2.0.0",
+      url: "https://esl.github.io/MongoosePush/v2.0.0/readme.html#content"
+    },
+    {
+      version: "v1.0.6",
+      url: "https://esl.github.io/MongoosePush/v1.0.6/readme.html#content"
+    }
+]
```

Comment:
```suggestion
]
```
Github shows that ugly `no-new-line` flag 😕